### ISSUE TITLE
Incorrect values returned for `@RRS`

### ISSUE THREAD

**Issue by NameOfTheDragon** [nexdome/Firmware#31]
Testing of pull request #30 has revealed a puzzling issue as follows:

```lang=txt
@FRS
FRS3.4.0-auto-close.5
@ZDS
ZDS
@RRS
RRS46000
@RWS,20000
RWS
@RRS
RRS200009
```

The final reading of `@RRS` returns the wrong value, always some extra data appended to the end.

Some thoughts on this:

- The XBee isn't implicated because it happens when issuing the commands locally on the shutter (as above).
- The command and payload appears to arrive intact when sent remotely from the rotator, as in this example:
  When sending `@RWS,20000` from the rotator, the shutter emits `Cmd @RWS,20000` which suggests that the data is arriving intact.
- Reading the configuration and performing microsteps-to-steps conversion doesn't appear to be implicated, because reverting to factory defaults restores the correct value, which it then reported correctly.
- The appended spurious data is always "09" so it doesn't seem to be a random memory corruption.

Conclusion: something is happening to the value between receiving it as a command and the value being written into the configuration structure.

This needs to be traced in detail to determine what's going wrong.

**Comment by NameOfTheDragon** [comment]
After reverting to factory defaults and resetting the target system, the problem can no longer be reproduced :(

**Comment by NameOfTheDragon** [comment]
Fixed by PR#30
